article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -54,13 +54,24 @@ case class ReaderRevenueLinks(
   sideMenu: ReaderRevenueLink
 )
 
+case class IsPartOf(
```

Comment:
How does this relate to the feature? (I am sure it does, just don't understand things well enough.)

Reply:
It's used by the Subscribe with Google extension to identify which product (productID) the user can subscribe/contribute to, and adds further metadata `["CreativeWork", "Product"]`, suggested by their docs.